src/pages/use-cases/gateway-api.jsx

```diff
@@ -45,13 +45,21 @@ const sectionContent3 = {
     'Cilium aims to simplify operations by reducing the number of cloud native networking, observability, and security tools required. It natively supports not just the Gateway API but also features like a sidecar-less Service Mesh, Transparent Encryption, Network Policies, and built-in observability with Hubble. This approach makes life easier for cluster operators, who only need a single tool to manage and upgrade for many, if not all, of their Kubernetes networking needs​​.',
   videoSrc: 'https://www.youtube.com/embed/48ej2qIP92Y',
 };
+const sectionContent4 = {
+  title: 'Gateway API for Service Mesh (GAMMA)',
+  imageRight: false,
+  description:
+    'By leveraging the standard Kubernetes Gateway API for Mesh Management and Administration(GAMMA), Cilium can be used for sophisticated East-West traffic management within the cluster. Cilium support for GAMMA translates into simplicity for operators while providing advanced traffic management.',
+  videoSrc: 'https://www.youtube.com/embed/OAftVdKllak',
```

Comment:
```suggestion
  videoSrc: 'https://youtu.be/DYenSJoEVwY?si=J-qKWqJcVM-IAnQG',
```

Reply:
I thought the general rule of thumb here is a preference for content from the Cilium community?

Reply:
good point, I was thinking about using the shorter video, but I can see your point